MANIFEST.in

```diff
@@ -1,3 +1,4 @@
+pruny terryfy
```

Comment:
Typo -- I think you meant `prune`.

Reply:
Yikes, how'd that happen. Thanks.

Reply:
`python setup.py sdist` outputs a warning:
```
reading manifest template 'MANIFEST.in'
warning: manifest_maker: MANIFEST.in, line 1: unknown action 'pruny'
```

But `check-manifest -v` does not.

Reply:
Yeah, it would be good if check-manifest would syntactic validity as well ->https://github.com/mgedmin/check-manifest/issues/85